pkg/cgroups/cgroups.go

```diff
@@ -0,0 +1,155 @@
+package cgroups
+
+import (
+	"bufio"
+	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// map of cgroup subsystems to their respective paths
+type Cgroups map[string]string
+
+// Find retrieves the mount points for all cgroup subsystems on the host. The
+// result of this operation should be cached if possible.
+func Find() (Cgroups, error) {
+	// refer to `man proc` or
+	// https://www.kernel.org/doc/Documentation/filesystems/proc.txt section 3.5
+	mountInfo, err := os.Open("/proc/self/mountinfo")
```

Comment:
The possibility of running on a system without cgroup support is interesting, I didn't think about it much at the time of writing this. For starters you probably wouldn't have cgexec on system so you'd be dead from that... is this worth supporting?